doc/src/modules/matrices/matrices.rst

```diff
@@ -331,24 +331,27 @@ applying ``gcd`` to the data of the matrix:
     >>> from sympy.abc import x, y
     >>> from sympy import gcd
     >>> m = Matrix([[x, y], [1, x*y]]).inv('ADJ'); m
-    [  x*y       -y    ]
+    [   x        -1    ]
+    [ ------    ------ ]
+    [  2         2     ]
+    [ x  - 1    x  - 1 ]
+    [                  ]
+    [  -1         x    ]
     [--------  --------]
     [ 2         2      ]
     [x *y - y  x *y - y]
+    >>> gcd(tuple(_))
+    1
```

Comment:
This should not be an intended result because this tutorial is actually about how to use GCDs to simplify the terms.

Reply:
To be clear I am still just using this PR as a way of experimenting with DomainMatrix. I won't ask for this to be merged directly. Rather when I am done I will take out the domain matrix parts and put them in a separate PR. Ordinary matrices would (at least initially) be left alone.